### PR TITLE
Use a masonry layout for the library layout

### PR DESCRIPTION
**Pull request by jacksonh**

### CODE REVIEW

**Comment by rupin90** [comment]
I am sorry I am lacking context here? Is there a ticket to this I can look into?

File: packages/web/package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "react-colorful": "^5.5.1",
     "react-dom": "^17.0.2",
     "react-hot-toast": "^2.1.1",
+    "react-masonry-css": "^1.0.16",
```
Comment (rupin90) [inline packages/web/package.json:46]: This is Badass stuff!!! 💯

Comment (jacksonh) [reply packages/web/package.json:46]: Yeah turned out really nice!

**Comment by jacksonh** [comment]
@rupin90 here's a little more detail. Mostly its just the new layout displayed in Figma: https://github.com/omnivore-app/omnivore/issues/1112

**Review by rupin90 — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**